core-bundle/contao/library/Contao/Input.php

```diff
@@ -545,9 +545,9 @@ private static function cleanKeyInternal($varValue)
 	/**
 	 * Strip HTML and PHP tags preserving HTML comments
 	 *
-	 * @param mixed  $varValue             A string or array
-	 * @param string $strAllowedTags       A string of tags to preserve
-	 * @param string $strAllowedAttributes A serialized string of attributes to preserve
+	 * @param mixed                                       $varValue             A string or array
+	 * @param string                                      $strAllowedTags       A string of tags to preserve
+	 * @param string|list<array{key:string,value:string}> $strAllowedAttributes A serialized string of attributes to preserve
```

Comment:
This change seems unrelated. What is it for?

Reply:
@ausi

Reply:
PHPStan fix because of https://github.com/contao/contao/blob/9cfdff0907612cba333bb82d1a8c1d20a2e00458/core-bundle/src/Twig/Runtime/SanitizerRuntime.php#L75

Reply:
We will encapsulate the change in #5682.